cpp/velox/compute/WholeStageResultIterator.cc

```diff
@@ -269,9 +269,6 @@ WholeStageResultIteratorFirstStage::WholeStageResultIteratorFirstStage(
       }
       task->noMoreSplits(scanNodeIds_[idx]);
     }
-    for (const auto& streamId : streamIds_) {
```

Comment:
@zzcclp @zhejiangxiaomai @rui-mo Could you explain why remove this?  We encountered `noMoreSplit` issue after remove `task->moMoreSplits`.

The full exception stack is 
```
Job aborted due to stage failure: Task 54 in stage 10.0 failed 50 times, most recent failure: Lost task 54.49 in stage 10.0 (TID 51268) (n211-139-219.byted.org executor 116): java.lang.RuntimeException: Exception: VeloxUserError
Error Source: USER
Error Code: INVALID_ARGUMENT
Reason: Splits can be associated only with leaf plan nodes which require splits. Plan node ID 0 doesn't refer to such plan node.
Retriable: False
Expression: splitPlanNodeIds_.find(id) != splitPlanNodeIds_.end()
Function: checkPlanNodeIdForSplit
File: /opt/tiger/compile_path/src/code.byted.org/olap/gluten/ep/build-velox/build/velox_ep/velox/exec/Task.cpp
Line: 851
Stack trace:
Stack trace has been disabled.Use --velox_exception_user_stacktrace=true to enable it.

	at io.glutenproject.vectorized.ArrowOutIterator.nativeHasNext(Native Method)
	at io.glutenproject.vectorized.ArrowOutIterator.hasNextInternal(ArrowOutIterator.java:48)
	at io.glutenproject.vectorized.GeneralOutIterator.hasNext(GeneralOutIterator.java:37)
	at io.glutenproject.backendsapi.glutendata.GlutenIteratorApi$$anon$2.hasNext(GlutenIteratorApi.scala:242)
	at io.glutenproject.vectorized.CloseableColumnBatchIterator.hasNext(CloseableColumnBatchIterator.scala:41)
	at org.apache.spark.InterruptibleIterator.hasNext(InterruptibleIterator.scala:37)
	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
	at org.apache.spark.shuffle.GlutenColumnarShuffleWriter.internalWrite(GlutenColumnarShuffleWriter.scala:96)
	at org.apache.spark.shuffle.GlutenColumnarShuffleWriter.write(GlutenColumnarShuffleWriter.scala:196)
	at org.apache.spark.shuffle.ShuffleWriteProcessor.write(ShuffleWriteProcessor.scala:59)
	at org.apache.spark.scheduler.ShuffleMapTask.runTask(ShuffleMapTask.scala:99)
	at org.apache.spark.scheduler.ShuffleMapTask.runTask(ShuffleMapTask.scala:52)
	at org.apache.spark.scheduler.Task.run(Task.scala:134)
	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$3(Executor.scala:514)
	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:1524)
	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:517)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
```

Reply:
Hi @WangGuangxin , this is not needed any more because`ArrowStreamNode` does not require splits.